app/templates/media/index.hbs

```diff
@@ -33,7 +33,8 @@
               }}
             </div>
             <div class="filter-legend">
-              <div class="low-value">1907</div>
+              <div class="low-value">1868
+              </div>
```

Comment:
Can you move this closing tag back up to the same line?

Reply:
@matthewdias done.